admin-dev/themes/default/template/header.tpl

```diff
@@ -364,16 +364,16 @@
             <li class="text-left text-nowrap username" data-mobile="true" data-from="employee_links" data-target="menu">{l|escape s='Welcome back %name%' sprintf=['%name%' => $employee->firstname] d='Admin.Navigation.Header'}</li>
             <li class="employee-wrapper-profile"><a class="admin-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}"><i class="material-icons">edit</i> {l|escape s='Your profile' d='Admin.Navigation.Header'}</a></li>
             <li class="divider"></li>
-            <li><a href="{l|escape s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17
-' d='Admin.Navigation.Header'}" target="_blank"><i class="material-icons">book</i> {l|escape s='Resources' d='Admin.Navigation.Header'}</a></li>
-            <li><a href="{l|escape s='https://www.prestashop.com/en/training?utm_source=back-office&utm_medium=profile&utm_campaign=training-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank"><i class="material-icons">school</i> {l|escape s='Training' d='Admin.Navigation.Header'}</a></li>
-            <li><a href="{l|escape s='https://www.prestashop.com/en/experts?utm_source=back-office&utm_medium=profile&utm_campaign=expert-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank"><i class="material-icons">person_pin_circle</i> {l|escape s='Find an Expert' d='Admin.Navigation.Header'}</a></li>
-            <li><a href="{l|escape s='https://addons.prestashop.com?utm_source=back-office&utm_medium=profile&utm_campaign=addons-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank"><i class="material-icons">extension</i> {l|escape s='PrestaShop Marketplace' d='Admin.Navigation.Header'}</a></li>
-            <li><a href="{l|escape s='https://www.prestashop.com/en/contact?utm_source=back-office&utm_medium=profile&utm_campaign=help-center-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank"><i class="material-icons">help</i> {l|escape s='Help Center' d='Admin.Global'}</a></li>
-            {if $host_mode}
-              <li><a href="https://www.prestashop.com/cloud/" target="_blank" rel="noopener noreferrer nofollow"><i class="material-icons">settings_applications</i> {l|escape s='My PrestaShop account' d='Admin.Navigation.Header'}</a></li>
-            {/if}
-            <li class="divider"></li>
+
+            {foreach from=$displayBackOfficeEmployeeMenu item=$menuItemModule name=displayBackOfficeEmployeeMenu}
+              {foreach from=$menuItemModule item=$menuItem}
```

Comment:
```suggestion
            {foreach from=$displayBackOfficeEmployeeMenu item=menuItemModule}
              {foreach from=$menuItemModule item=menuItem}
```

The item parameter doesn't need a `$` in the foreach instruction...
And I think `name=displayBackOfficeEmployeeMenu` is not necessary here, since you don't use it 

https://www.smarty.net/docsv2/en/language.function.foreach.tpl

Reply:
Fixed